datadog_checks_base/datadog_checks/base/stubs/_util.py

```diff
@@ -10,7 +10,7 @@ class SubprocessOutputEmptyError(Exception):
     pass
 
 
-def subprocess_output(command, raise_on_empty_output):
+def subprocess_output(command, raise_on_empty_output, env=None):
```

Comment:
It cleaner to have a separate PR for things changing `datadog_checks_base`

Reply:
I’ve extracted the `datadog_checks_base` changes in #7479.
Once #7479 will be merged on master and this PR will be rebased on master, only the `network` changes will remain in this PR.